data/console/nginx_vhost.conf

```diff
@@ -0,0 +1,30 @@
+server {
+  listen 80;
```

Comment:
```suggestion
  listen 80 default_server;
```

Reply:
Why `default_server`? There is already one entry in `nginx.conf`.

Reply:
![image](https://user-images.githubusercontent.com/1254493/150756219-afbc142f-d010-4c88-ab55-54d726cfadf4.png)
In the `/etc/nginx/nginx.conf` there is:
```
    server {
        listen       80;
        server_name  localhost;
```

Reply:
Solved by defining the `default_server` as @grisu48 suggested.